app/boards/arm/preonic/Kconfig.defconfig

```diff
@@ -0,0 +1,17 @@
+# Preonic keyboard configuration
+
+# Copyright (c) 2022 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+if BOARD_PREONIC_REV3
+
+config ZMK_KEYBOARD_NAME
+	default "Preonic V3"
+
+config ZMK_USB
+	default y
```

Comment:
Ditto, there's really no reason not to enable USB for an onboard keyboard, so move to _defconfig file as well.

Reply:
Understood. Will look at it tonight. Many thanks for the notes :)

Reply:
Done